website_form_require_legal/readme/USAGE.rst

```diff
@@ -0,0 +1,10 @@
+To use this module, you need to:
+
+* Edit any page in your website.
+* Add a form from the snippets.
+* Modify the parameters.
+* You will find the option "Legal terms acceptance".
+* If you select it you will be able to modify the text shown and the link to the
+  page from the text editor.
+
+Note: Remember to change the terms and conditions link to redirect to the desired page.
```

Comment:
Should we add a ROADMAP declaring that this module is aimed to deprecate website_form_legal and its descendants?

Reply:
Added :smile: